### PR TITLE
Zmoji

### PR DESCRIPTION
**Pull request by scarolan**
I know you're not accepting themes, but can you make an exception for this one?  The Zmoji theme is meant to demonstrate how to use the zmoji plugin.

### CODE REVIEW

**Comment by CodeFalling** [comment]
It looks fancy! +1 for this.

**Comment by apjanke** [comment]
This PR is 22 commits long and contains unrelated-seeming changes to several other plugins, like `vi-mode`, `battery`, `bundler`, `cabal`, and `rake`. The changes to `gitignore` and `bundler` include unresolved merge conflicts. Did this PR accidentally pick up some extra changes? Could it be cleaned up, rebased, and the commits squashed to a shorter history?

File: plugins/moonphase/moonphase.plugin.zsh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#                moon emoji to appear.
+#        AUTHOR: Sean Carolan (scarolan[at]gmail.com)
+#       VERSION: 1.0.0
+# -----------------------------------------------------------------------------
+
+function printmoon() {
+  # Accepts a time in days between 0 and 29.53
+  # Defaults to the current moon phase
+  if (($+1)); then
+    phase=$1
```
Comment (apjanke) [inline plugins/moonphase/moonphase.plugin.zsh]: This and the other variables in `printmoon()` and `calcphase()` look like they should be `local`.

**Comment by apjanke** [comment]
The `zmoji` plugin looks redundant with the existing [`emoji` plugin](https://github.com/robbyrussell/oh-my-zsh/tree/master/plugins/emoji). Are there features that `emoji` doesn't support? I think we'd probably prefer to stick with a single plugin for emoji support.

**Pull request closed without merging**